python/cuml/cuml/fil/README.md

```diff
@@ -1,79 +1,79 @@
 # FIL - RAPIDS Forest Inference Library
```

Comment:
Is the content in here for users or for developers? If users, we should probably make an effort to move this to docs instead of a readme nested within the repo. Fine to merge as is, just a note.

Reply:
It sounds user-targeted to me and we should probably move it before merge.

Reply:
I think that should be done in a follow-up. The READMEs have been here for a while, so continuing with docs-in-readmes isn't making things worse, it's just maintaining the same. And afaik we _do_ want to get this PR in for 25.04. Happy to push a follow-up moving this content to docs, but I'd like to get this merged first.

Reply:
I've created https://github.com/rapidsai/cuml/issues/6492 to track.